samples/UWP/GravatarMobile-UWP.sln

```diff
@@ -11,17 +11,14 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GravatarMobile.UWP", "..\..
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
```

Comment:
Any CPU is not a valid configuration for UWP apps - it gets compiled to native code